packages/compiler/src/render3/r3_template_transform.ts

```diff
@@ -80,11 +80,21 @@ class HtmlAstToIvyAst implements html.Visitor {
   errors: ParseError[] = [];
   styles: string[] = [];
   styleUrls: string[] = [];
+  private inI18nBlock: boolean = false;
 
   constructor(private bindingParser: BindingParser) {}
 
   // HTML visitor
   visitElement(element: html.Element): t.Node|null {
+    const isI18nRootElement = isI18nRootNode(element.i18n);
+    if (isI18nRootElement) {
+      if (this.inI18nBlock) {
+        this.reportError(
+            'Could not mark an element as translatable inside of a translatable section',
```

Comment:
I think the message might be even clearer if we explicitly say that the "i18n" marker is the problem and add a call to action.

```suggestion
            'Cannot mark an element as translatable inside of a translatable section. Please remove the nested i18n marker.`
```

Reply:
Thanks for the review @kara, the message text is now updated.